### DISCUSSION TITLE
Is that possible to login into KIbana directly without login in Security onion please suggest

### DISCUSSION THREAD

**Discussion by sandraimmaculate** [Security-Onion-Solutions/securityonion#14012] [Unsupported Versions]
Hello, I am using Security Onion 2.4 with a 250GB root volume, 16GB RAM, and 4 vCPUs. I want to log in to Kibana directly via a separate URL, as currently, we have to log in to Security Onion first and then log in to Kibana. How can I log in directly to Kibana?. I have allowed the port 5601 for my network ip 
I'm able to connect using telnet. But not in browser why ?? 

![image](https://github.com/user-attachments/assets/c078e71c-bb35-4296-8936-969de7d5dd55)

Please suggest how to access kibana login page directly without login in security onion

**Comment by TOoSmOotH** [comment]
Nginx protects everything and the only way to get through nginx is to authenticate to SOC.

**Reply by sandraimmaculate** [reply]
Thanks for your reply !
Is it possible to configure Nginx for a direct URL to Kibana???. However, any modifications made to the Nginx configuration file are overwritten by the old configuration after a restart.
